### PR TITLE
Add french readme + fixes

### PR DESCRIPTION
**Pull request by Thovi98**

### CODE REVIEW

**Comment by Thovi98** [comment]
Oww I see I did not take te latest version of the Readme, I will update soon.

**Comment by Thovi98** [comment]
So now I think it’s okay, don’t hesitate to check if I let some mistakes.

File: README.md [hunk 1]
```diff
@@ -53,14 +61,14 @@ The admin interface is accessible at the address: `your.domain.fr/api/admin`
...
- * Report a bug about Funkwhale itself: https://code.eliotberriot.com/funkwhale/funkwhale/
  * App website: https://docs.funkwhale.audio
  * Upstream app repository: https://dev.funkwhale.audio/funkwhale/funkwhale
  * YunoHost website: https://yunohost.org/
 
 ---
 
-## Developers info
+Developers info
+----------------
```
Comment (ericgaspar) [inline README.md:71]:
```suggestion
## Developers info
```
Same result, but to keep consistency

File: README_fr.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+ * Signaler un bug : https://github.com/YunoHost-Apps/funkwhale_ynh
+ * Site de l’application : https://docs.funkwhale.audio
+ * Dépôt de l'application principale : https://dev.funkwhale.audio/funkwhale/funkwhale
+ * Site web YunoHost : https://yunohost.org/
+
+---
+
+## Informations pour les développeurs
+
+Merci de faire vos pull request sur la [branche testing](https://github.com/YunoHost-Apps/REPLACEBYYOURAPP_ynh/tree/testing).
```
Comment (ericgaspar) [inline README_fr.md:74]:
```suggestion
Merci de faire vos pull request sur la [branche testing](https://github.com/YunoHost-Apps/funkwhale_ynh/tree/testing).
```

**Review by ericgaspar — CHANGES_REQUESTED** [review]

**Review by ericgaspar — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -3,6 +3,8 @@
 [![Integration level](https://dash.yunohost.org/integration/funkwhale.svg)](https://dash.yunohost.org/appci/app/funkwhale) ![](https://ci-apps.yunohost.org/ci/badges/funkwhale.status.svg) ![](https://ci-apps.yunohost.org/ci/badges/funkwhale.maintain.svg)  
 [![Install Funkwhale with YunoHost](https://install-app.yunohost.org/install-with-yunohost.png)](https://install-app.yunohost.org/?app=funkwhale)
```
Comment (ericgaspar) [inline README.md:4]:
```suggestion
[![Install Funkwhale with YunoHost](https://install-app.yunohost.org/install-with-yunohost.svg)](https://install-app.yunohost.org/?app=funkwhale)
```

File: README_fr.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
+# Funkwhale pour YunoHost
+
+[![Integration level](https://dash.yunohost.org/integration/funkwhale.svg)](https://dash.yunohost.org/appci/app/funkwhale) ![](https://ci-apps.yunohost.org/ci/badges/funkwhale.status.svg) ![](https://ci-apps.yunohost.org/ci/badges/funkwhale.maintain.svg)   
+[![Install Funkwhale with YunoHost](https://install-app.yunohost.org/install-with-yunohost.png)](https://install-app.yunohost.org/?app=funkwhale)
```
Comment (ericgaspar) [inline README_fr.md:4]:
```suggestion
[![Install Funkwhale with YunoHost](https://install-app.yunohost.org/install-with-yunohost.svg)](https://install-app.yunohost.org/?app=funkwhale)
```

**Review by ericgaspar — CHANGES_REQUESTED** [review]
Let's change "Install with YunoHost" badge to SVG version

File: README_fr.md (hunk 4 above)
Comment (Thovi98) [reply README_fr.md]: Yes, it’s far better !!

**Review by ericgaspar — APPROVED** [review]

**Review by Sporiff — APPROVED** [review]

**Pull request merged**